.github/workflows/build-test.yml

```diff
@@ -85,15 +46,17 @@ jobs:
         run: cd hosting && rm -f .env && yarn build --mode=${{ env.BUILD_ALIAS }}
         env:
           NODE_ENV: production
-          VUE_APP_FIREBASE_API_KEY: ${{ secrets[env.SECRET_NAME_FIREBASE_API_KEY] }}
-          VUE_APP_FIREBASE_APP_ID: ${{ secrets[env.SECRET_NAME_FIREBASE_APP_ID] }}
-          VUE_APP_FIREBASE_AUTH_DOMAIN: ${{ secrets[env.SECRET_NAME_FIREBASE_AUTH_DOMAIN] }}
-          VUE_APP_FIREBASE_DATABASE_U_R_L: ${{ secrets[env.SECRET_NAME_FIREBASE_DATABASE_U_R_L] }}
-          VUE_APP_FIREBASE_MESSAGING_SENDER_ID: ${{ secrets[env.SECRET_NAME_FIREBASE_MESSAGING_SENDER_ID] }}
-          VUE_APP_FIREBASE_PROJECT_ID: ${{ secrets[env.SECRET_NAME_FIREBASE_PROJECT_ID] }}
-          VUE_APP_FIREBASE_STORAGE_BUCKET: ${{ secrets[env.SECRET_NAME_FIREBASE_STORAGE_BUCKET] }}
-          VUE_APP_FUNCTIONS_REGION: ${{ secrets[env.SECRET_NAME_FUNCTIONS_REGION] }}
-          VUE_APP_BUILD_VERSION: ${{ env.BUILD_VERSION }}
+          BUILD_ALIAS: production
```

Comment:
For `vue-cli-service build`, the default mode is `production`.

...and I don't think we have *any* code that's dependent on mode. It only drives which `.env` files to pick up; but here you delete `.env` & set it explicitly via CLI.

TL;DR: You can delete the `BUILD_ALIAS` & remove `--mode...` if you want.